index.html

```diff
@@ -3,8 +3,8 @@
   <head>
     <title>Scratch Empires Home</title>
   </head>
-  <body Backgorund-image: url("https://raw.githubusercontent.com/Autofirejm/Scratch-Empires/main/2022-01-20_13.43.01.png")>
-    <H1 style="text-align:center; color:"#ffffff""> Scratch Empires Website</H1>
+  <body backgorund-image: url("https://raw.githubusercontent.com/Autofirejm/Scratch-Empires/main/2022-01-20_13.43.01.png")>
```

Comment:
Still misspelled...

Reply:
Hey @ILOVETOMAKESTUFF I have fixed that.